compiler/src/dotty/tools/repl/ReplDriver.scala

```diff
@@ -255,8 +255,22 @@ class ReplDriver(settings: Array[String],
     else
       label
 
+  @deprecated("Use completionsWithSignatures instead", "3.4.2")
+  protected final def completions(cursor: Int, expr: String, state0: State): List[Candidate] =
```

Comment:
Is it `java.util.List` or `scala.List`. These were previously passed directly to the jline  methods expecting `java.util.Collection[? <: org.jline.reader.Candidate]` 

I assume it's `java.util.List`, but just want to be sure